internal/tutils/tutils.go

```diff
@@ -20,9 +21,9 @@ var AuthString1 = GetCompleteBase64Header("000001")
 
 var FedAuth = getBase64Header(fedoraHeader, "User")
 
-func GetResponseError(url string) (*http.Response, error) {
+func GetResponseError(addr, path string) (*http.Response, error) {
 	client := &http.Client{}
-	request, err := http.NewRequest("GET", url, nil)
+	request, err := http.NewRequest("GET", fmt.Sprintf("http://%s%s", addr, path), nil)
```

Comment:
fwiw, this is not super-great if we want to do https but (afaict) we don't test it right now and it seems easy enough to tweak once we do